jerry-core/api/jerry.c

```diff
@@ -396,10 +396,17 @@ jerry_parse (const jerry_char_t *resource_name_p, /**< resource name (usually a
 #if ENABLED (JERRY_PARSER)
   jerry_assert_api_available ();
 
-#if ENABLED (JERRY_LINE_INFO)
-  JERRY_CONTEXT (resource_name) = ecma_find_or_create_literal_string (resource_name_p,
-                                                                      (lit_utf8_size_t) resource_name_length);
-#endif /* ENABLED (JERRY_LINE_INFO) */
+#if ENABLED (JERRY_LINE_INFO) || ENABLED (JERRY_ERROR_MESSAGES)
+  if (!resource_name_length)
```

Comment:
Probably it's only my personal preference, but could you please use `resource_name_length == 0` instead?

Reply:
Sure.